scripts/ci/sizes.py

```diff
@@ -109,28 +109,26 @@ def compare(previous_path: str, current_path: str, threshold: float) -> None:
             entries[name] = {}
         entries[name]["previous"] = entry
 
-    headers = ["Name", "Previous", "Current", "Change"]
+    headers = ["Name", before_header, after_header, "Change"]
     rows: list[tuple[str, str, str, str]] = []
     for name, entry in entries.items():
         if "previous" in entry and "current" in entry:
             previous = float(entry["previous"]["value"]) * DIVISORS[entry["previous"]["unit"]]
             current = float(entry["current"]["value"]) * DIVISORS[entry["current"]["unit"]]
 
-            min_change = previous * (threshold / 100)
-
             unit = get_unit(min(previous, current))
             div = get_divisor(unit)
 
-            change = ((previous / current) * 100) - 100
-            sign = "+" if change > 0 else ""
+            change_pct = ((current - previous) / previous) * 100
 
+            min_change = previous * (threshold / 100)
```

Comment:
The units here confuse me. What unit does `threshold` have?  If it is in percents, why not just compare `abs(change_pct) >= threshold` ? Again: proper names help a lot.

I prefer to not use percentages except for ui:s (i.e. multiply with 100 during the formatting). It usually ends up with far less `100` in the code (and less bugs). `_ratio` or `_fraction` is a pretty good suffix for such numbers. Percentages in arguments have the problem of the unit being invisible. `--threshold 0.05` is usually clearer than `--threshold 5` (5 kiB?)

Reply:
If you're about to use the script, you're going to run `sizes.py --help` first, and the help text for `threshold` says `N%`, so `--threshold 5` means `5%`. As a user, that seems marginally better than `0.05`, because the table also outputs percentages. I tell the script I want to see differences of more than 5% and I see all the differences which were more than 5%, no mental math required, however little it may be.